python/chemiscope/jupyter.py

```diff
@@ -115,6 +119,7 @@ def show_input(path, mode="default"):
     :param str path: load the chemiscope widget from path.
 
     :param str mode: widget mode, either ``default``, ``structure`` or ``map``.
+    :param float warning_timeout: timeout (in ms) for warnings
```

Comment:
The same doc that exists in `show` should be duplicated everywhere this parameter exists, otherwise people might not understand what this parameter is for

Reply:
@ceriottm still relevant